core/frontend/src/render/webgl/glsl/Edge.ts

```diff
@@ -27,11 +27,20 @@ export type EdgeBuilderType = "SegmentEdge" | "Silhouette" | "IndexedEdge";
 
 const computeOtherPos = `
   vec2 tc = computeLUTCoords(g_otherIndex, u_vertParams.xy, g_vert_center, u_vertParams.z);
-  vec4 enc1 = floor(TEXTURE(u_vertLUT, tc) * 255.0 + 0.5);
-  tc.x += g_vert_stepX;
-  vec4 enc2 = floor(TEXTURE(u_vertLUT, tc) * 255.0 + 0.5);
-  vec3 qpos = vec3(decodeUInt16(enc1.xy), decodeUInt16(enc1.zw), decodeUInt16(enc2.xy));
-  g_otherPos = unquantizePosition(qpos, u_qOrigin, u_qScale);
+  if (g_usesQuantizedPosition) {
```

Comment:
@DStradley @MarcNeely `g_usesQuantizedPosition` is true if `u_qScale.x` is less than zero.
Do you expect the compiler to be smart enough to optimize based on that, or should there be a uniform `u_usesQuantizedPositioni` instead?

Reply:
I expect the D3D compiler in Angle to be smart enough to optimize this, we could look at shader code to be sure I guess.  I would hope that others would too, but don't know.

Reply:
I don't know.  You could probably compare the before & after compiled shader output differences & see if it did.

Reply:
Unfortunately I do not know how to interpret the shader output.

Reply:
Did you run a test with this on ImageTests, if so, David & I could look at the shaders from that.

Reply:
The skip shaders flag is on by default (to save a few minutes), so they would not have built on his branch.  I'll try pulling and building them.

Reply:
We looked at the Angle(D3D) shader assembly and it is optimized fine like this, as I expected.  After thinking about it some more, other compilers should surely handle this fine as well.